### DISCUSSION TITLE
Kepler release 0.4 planning

### DISCUSSION THREAD

**Discussion by rootfs** [sustainable-computing-io/kepler#335] [General]
The goal of release 0.4 is to have complete end-to-end integration of Operator, Kepler, model server, and estimator, with sufficient test cover and documentation, so that the project can be deployed in its entirety. 

Please add your comments into the project board and linked issues. We'll have a 1-hour meeting in early Nov to go through the size, review and assignment. Please go to the poll in slack #project channel book your referred meeting time. 
https://github.com/orgs/sustainable-computing-io/projects/2

**Comment by marceloamaral** [comment]
Thanks @rootfs, this is a very good plan! I left some comments on two tasks.
